tests/test_local_persistence.py

```diff
@@ -0,0 +1,40 @@
+import random
+import tempfile
+
+import numpy as np
+
+import qdrant_client
+import qdrant_client.http.models as rest
+
+default_collection_name = "example"
+
+
+def ingest_data(
+    vector_size=1500, path=None
+):  # vector_size < 433: works, vector_size >= 433: crashes
+    lines = [x for x in range(10)]
+
+    embeddings = np.random.randn(len(lines), vector_size).tolist()
+    client = qdrant_client.QdrantClient(path=path)
+
+    client.recreate_collection(
+        default_collection_name,
+        vectors_config=rest.VectorParams(
+            size=vector_size,
+            distance=rest.Distance.COSINE,
+        ),
+    )
+
+    client.upsert(
+        collection_name=default_collection_name,
+        points=rest.Batch.construct(
+            ids=[random.randint(0, 100) for _ in range(len(lines))],
+            vectors=embeddings,
+        ),
```

Comment:
Sometimes that may lead to duplicated ids in the collection. Since the test doesn't check how many points were stored, that wasn't visible. It should be instead changed to:

```python
points=rest.Batch.construct(
    ids=random.sample(range(100), len(lines)),
    vectors=embeddings,
),
```